extensions/amp-brightcove/0.1/validator-amp-brightcove.protoascii

```diff
@@ -36,6 +36,13 @@ tags: {  # <amp-brightcove>
     name: "data-account"
     mandatory: true
   }
+  # <amp-bind>
+  attrs: { "[data-account]" }
```

Comment:
This isn't going to quite work the way you want it. Since data-account is mandatory, a tag like:

<amp-brightcove [data-account]="..."> will not validate, since the "data-account" attribute is missing.

The best solution would be one that is aware of bind attributes, but working within the current rules system, I think we should use a rarely-used rule feature to do something like:

attrs: {
  name: "data-account"
  alternative_names: "[data-account]"
  mandatory: true
}

And then leave off the separate attribute. Now that I think about it, this might generally be a good approach for these bind attributes for now.